### ISSUE TITLE
branch master checkout, but  mvn  clean install error 

### ISSUE THREAD

**Issue by guangdashao** [mattcasters/kettle-neo4j-core#12]
mvn clean install error:

`[ERROR]   The project kettle-neo4j:kettle-neo4j-core:2.0.5 (D:\ChromeCoreDownloads\kettle-neo4j-core-master\kettle-neo4j-core-master\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not transfer artifact org.pentaho:pentaho-ce-jar-parent-pom:pom:8.2.0.7-719 from/to central (https://repo.maven.apache.org/maven2): Connect to repo.maven.apache.org:443 [repo.maven.apache.org/151.101.196.215] failed: Connection timed out: connect and 'parent.relativePath' points at wrong local POM @ line 5, column 11 -> [Help 2]`

and on https://mvnrepository.com website, i can't find `<groupId>org.pentaho</groupId>
    <artifactId>pentaho-ce-jar-parent-pom</artifactId>
    <version>8.2.0.7-719</version>`

**Comment by mattcasters** [comment]
Fair enough. I updated the READ.md file:  https://github.com/mattcasters/kettle-neo4j-core/blob/master/README.md
